html-and-the-web/html-and-the-web_div-salad/index.html

```diff
@@ -13,59 +13,59 @@
   </head>
   <!--someone forgot to wrap the content of our webpage with an HTML element,
 	which one was it again?-->
-  <div>
-    <div>
-      <div>Jane Doe</div>
+  <body>
+    <header>
+      <h1>Jane Doe</h1>
       <!--Which elements could be used to link to parts of the webpage,
 		which appear below?-->
-      <div id="nav">
-        <div>About Me</div>
-        <div>Contact</div>
-        <div>Skillset</div>
-      </div>
-    </div>
+      <nav id="nav">
+        <a href="#">About Me</a>
```

Comment:
Please correct the href element to : a href="#about-me" and similarly for the contact and skillset id elements.